source/_integrations/airnow.markdown

```diff
@@ -29,3 +29,7 @@ To generate an AirNow API key, go to the [AirNow Developer Tools Page](https://d
 The AirNow API allows 500 data updates per hour, but since observations are only updated hourly, the default update rate is set to 2 per hour and should not trigger rate limiting. If you use this API key for other purposes, ensure the total request rate does not exceed 500 per hour.
 
 </div>
+
+## Forecasts
+
+You may optionally choose to also retrieve air quality forecasts. This may be enabled during integration setup. Forecasts will be included as an attribute on the AQI sensor if this is enabled.
```

Comment:
As per review comment on the parent PR, these should not be added as attributes to sensors. In general, we want to avoid using attributes as much as possible. In this case, a `weather` entity might be more fitting.

Marking this PR for needing changes, considering we expect the parent PR to need changes that affects this documentation.